### ISSUE TITLE
Fullscreen on iOS doesn't work as expected

### ISSUE THREAD

**Issue by ayubov** [vimeo/player.js#624]
Hello!

### Expected Behavior
Player plays fullscreen with native controls in Safari

### Actual Behavior
Player doesn't play fullscreen, no native controls are visible

### Description
I've spent quite a lot of time on this issue and still don't understand how to fix it. 

What I'm doing is: 
1) Create instance of VimeoPlayer with following options `
{
  background: false,
  autoplay: false,
}
`
2) Call `.requestFullscreen()` and `.play()` methods on it inside on onclick handler

### Demo
Here is a demo how it looks with the same app on Android and iOS (both new versions):
https://yadi.sk/i/gyeM7dKPqJJ8ig

Please help me :)

**Comment by luwes** [comment]
Try setting `inline: false` as an option. that should play the the video in fullscreen on mobile.

No need for using the requestFullscreen api.

**Comment by ayubov** [comment]
> Try setting `inline: false` as an option. that should play the the video in fullscreen on mobile.
> 
> No need for using the requestFullscreen api.

Thank you, @luwes, I added `playsinline: false` as an option and it is better now. I can see native controls. But still after I call `play()` method on the video, one extra click is required (https://yadi.sk/i/Ji2v9qnEsfqj0w). Is there any way to avoid that extra click?

**Comment by luwes** [comment]
is it throwing any errors in the console?

if it's throwing a NotAllowed error or something with user gesture required it's possible you might have to set the video to muted initially. add the `muted: true` option.

did you add the correct iframe attributes `allow="autoplay"`?

**Comment by ayubov** [comment]
Yes, it was throwing `The request is not allowed by the user agent or the platform in the current context, possibly because the user denied permission.`.

Uh, I found a solution. The problem was that I didn't satisfy a requirement of “as a result of a user gesture” that is required by Safari. I fired `play()` not in the click event's handler, but later asynchronously. Now I fixed that and it works! 🎉

Here is an article where I figured out my problem: https://webkit.org/blog/6784/new-video-policies-for-ios/
